UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -313,13 +317,13 @@ private void InitializeEnvironment()
 
             SetIsInference(!IsCommunicatorOn);
 
-            BrainDecideAction += () => {};
-            DestroyAction += () => {};
-            AgentSetStatus += i => {};
-            AgentResetIfDone += () => {};
-            AgentSendState += () => {};
-            AgentAct += () => {};
-            AgentForceReset += () => {};
+            BrainDecideAction += () => { };
```

Comment:
these spaces keep flip flopping. Maybe we should include an `.editorconfig` file in the repo.